kvstore/src/io_utils.rs

```diff
@@ -43,6 +43,11 @@ pub struct CRCReader<R: Read> {
     chunk_size: usize,
 }
 
+/// Helper trait to make zeroing buffers easier
+pub trait FillExt<T> {
```

Comment:
nit: can we find a better name than `Ext` here?  `FillValue` perhaps?  There's an existing `Fill` trait that you're trying to avoid conflicting with?

Reply:
I'll change it to `Fill`